src/clojure_sandbox/4clojure.clj

```diff
@@ -0,0 +1,182 @@
+(ns clojure-sandbox.4clojure
+  (:require [clojure.string]))
```

Comment:
Maybe alias is? 

```suggestion
  (:require [clojure.string :as str]))
```

That way you don't have to use the entire namespace for using the fns

`clojure.string/join` -> `str/join`

Reply:
Thanks @yudistrange, done!